### DISCUSSION TITLE
OneDrive有一个程序自动创建的文件夹是什么意思

### DISCUSSION THREAD

**Discussion by StudenGit** [AlistGo/alist#1682] [General]
如题，为什么会有这么一个奇怪的东西？

**Comment by ChirmyRam** [comment]
那是你用了E5 RenewX网页端或者RenewPlus客户端后产生的，建议拿个不常用的空白子号来挂并及时清理文件，与alist无关。

**Comment by anwen-anyi** [comment]
楼上正解，就是 **E5 RenewX** 创建的。

**Comment by cancancancancancancan** [comment]
我这也有，同用了RenewPlus客户端，直接在alist中隐藏就好了，眼不见为净
